Dockerfile

```diff
@@ -0,0 +1,33 @@
+# define base image
+FROM node:lts
+
+# ENV NODE_ENV=production
+ENV YARN_VERSION 1.22.17
+RUN yarn policies set-version $YARN_VERSION
+
+# define working directory inside the container
+WORKDIR /usr/src/app
+
+# Copy the package.json file to the project directory
+COPY package.json .
+COPY yarn.lock .
+COPY .env .
+
+# install all the dependencies
+RUN yarn --frozen-lockfile
+
+# Copy all the source code from host machine to the container project directory
+COPY . .
+
+RUN yarn build
+
+COPY .env bin/
+
+# change directory to the bin diretory
```

Comment:
Thanks for the documentation.  It is really important to document your Dockerfile to indicate what changes are happening outside the container vs inside the container!